### PR TITLE
Make ember-load-initializers an addon

### PR DESCRIPTION
**Pull request by josemarluedke**
Hello there. 

This is an attempt to migrate to addon. Not sure if everything is in the best way to do things, but I'm looking for some feedbacks and I will fix it. In my tests, it works fine with changes required in ember-cli (PR coming as well).

Helps with ember-cli/ember-cli#4546.

### CODE REVIEW

**Comment by rwjblue** [comment]
This looks good! Thanks for working on it!

Can you add a shim for the old module ('ember/load-initializers')? The ember-resolver addon did something similar for reference (vendor file + app.import I think).

File: index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* jshint node: true */
+'use strict';
+
+module.exports = {
+  name: 'ember-load-initializers',
+
+  included: function() {
+    this._super(...arguments);
```
Comment (rwjblue) [inline index.js]: This executes in Node, where we may not have spread arguments.

I'd suggest using `this._super.apply(this, arguments);` instead.

**Comment by josemarluedke** [comment]
@rwjblue Just added the shims. We should be good now.

**Comment by rwjblue** [comment]
Thank you @josemarluedke!!!

**Comment by rwjblue** [comment]
```
% npm publish
+ ember-load-initializers@0.5.0
% npm owner add stefanpenner
+ stefanpenner (ember-load-initializers)
```

**Pull request merged**